src/vue-testing-library.js

```diff
@@ -36,7 +36,7 @@ function render(
   }
 
   if (routes) {
-    const VueRouter = require('vue-router')
+    const VueRouter = require('vue-router') || require('vue-router').default
```

Comment:
The code is ugly cause the problem is that  in mocha unit tests we need to bundle the code with webpack (mocha-webpack) and that does not play well with using a require here. 

The problem could be also solved with an async import (to follow ES6 notation) but that would probably make the code even uglier cause it would introduce async code.

Another workaround when using mocha, would be passing an already built router through `router` mounting option (instead  of `routes`)  but that cannot work with the current implementation.

Enabling this option for mocha users would be highly appreciated. Thanks for your awesome work!